src/main/kotlin/transaction/TradeTransaction.kt

```diff
@@ -0,0 +1,37 @@
+package transaction
+
+import currency.CryptoCurrency
+import wallet.Wallet
+import java.math.BigDecimal
+import java.time.LocalDateTime
+import java.util.*
+
+class TradeTransaction(
+    id: UUID,
+    date: LocalDateTime,
+    initiator: Wallet,
+    fromCurrency: CryptoCurrency,
+    fromAmount: BigDecimal,
+    val receiver: Wallet,
+    toCurrency: CryptoCurrency,
+    toAmount: BigDecimal
```

Comment:
Use trailing comma in such classes (dto or models). You can make your commits cleaner if you will add new fields